docs/victory-area/docs.md

```diff
@@ -139,10 +139,19 @@ The `domainPadding` prop specifies a number of pixels of padding to add the begi
 
 ### label
 
+**DEPRECATED** use `labels` instead
```

Comment:
so is `label` now simple an alias for `labels`? if so, maybe we should just say that (a la the lodash docs with aliases or renaming)